sass/themes/cr/2017/components/navigation/_expanded-footer.scss

```diff
@@ -0,0 +1,292 @@
+// Expanded Footer
+//
+// Expanded footer (aka TURBOFOOTER™) for the CR.com site in 2018
+// 
+// Markup: expanded-footer.twig
+//
+// Style guide: 185.5
+
+footer.footer--expanded {
+  
+  background: #f2f2f2;
+  padding: 45px 18px 40px;
+
+  span.lg-breakpoint {
+    display: none;
+    visibility: hidden;
+  }
+
+  @include breakpoint($screen-lg) {
+    padding: 65px 18px 60px;
+
+    span.lg-breakpoint {
+      display: block;
+    }
+  }
+
+  // Social Links
+  .footer__social-links {
+    margin: 0 0 40px;
+    ul li {
+      margin: 5px 8px 0;
+      a {
+        width: 52px;
+        height: 52px;
+        background: none;
+      }
+    }
+
+    @include breakpoint($screen-md) { 
+      ul li {
+        margin: 5px 10px 0;
+        a {
+          width: 75px;
+          height: 75px;
+        }
+      }
+    }
+
+    @include breakpoint($screen-lg) {
+      margin: 0 0 75px;
+    } 
+  }
+
+  .footer__copyright {
+    margin-top: 10px;
+
+    @include breakpoint($screen-md) {
+      margin-top: 15px;
+    }
+
+    @include breakpoint($screen-lg) {
+      margin-top: 10px;
+    }
+    
+    p {
+      color: $colour-black;
+      font-size: 18px;
+      line-height: 29px;
+    }
+  } 
+
+  .footer__branding {
+    margin-top: 35px;
+
+    a {
+      width: 85px;
+      height: 85px;
+    }
+    
+    @include breakpoint($screen-md) {
+      margin-top: 35px;
+    }
+    
+    @include breakpoint($screen-lg) {
+      margin-top: 60px;
+    }
+  }
+
+  .region.region-footer.cr-footer {
+    @include container();
+  }
+
+  ul.menu.menu--level-0:not(.menu--level-1) {
+    width: 100%;
+    display: -webkit-flex;
+    display: -moz-flex;
+    display: -ms-flex;
+    display: -o-flex;
+    display : flex;
+    -webkit-flex-wrap: wrap;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+
+     // Represents each submenu/category in the footer
+     >li.menu-item {
+      @include span(12);
+      margin-bottom: 30px;
+      text-align: center;
+
+       @include breakpoint($screen-md) {
+         @include span(6);
+         margin-bottom: 25px;
+         text-align: left;
+
+         &.menu-item--1,
+         &.menu-item--3,
+         &.menu-item--5 {
+          @include span(5);
+          @include push(0.5);
+          text-align: right;
+          // Space for the chevron
+          // padding-right: 23px;
+         }
+
+        &.menu-item--2,
+         &.menu-item--4,
+         &.menu-item--6 {
+          @include span(5);
+          @include push(1);
+         }
+       }
+
+       @include breakpoint($screen-lg) {
+        &.menu-item--1,
+        &.menu-item--2,
+        &.menu-item--3,
+        &.menu-item--4,
+        &.menu-item--5,
+        &.menu-item--6 {
+          @include span(4);
+          text-align: left;
+          // padding-right: 0;
```

Comment:
should we not remove this?

Reply:
Ooh yeah right, probably have a little bit of housecleaning to do; will update